test/unittest/dds/subscriber/DataReaderTests.cpp

```diff
@@ -2446,60 +2437,6 @@ TEST_F(DataReaderUnsupportedTests, UnsupportedDataReaderMethods)
                 query_parameters));
     }
 
-    {
-        EXPECT_EQ(
-            ReturnCode_t::RETCODE_UNSUPPORTED,
-            data_reader->delete_readcondition(nullptr));
```

Comment:
Remove delete_readcondition from the list of unimplemented methods in this test comment.

Reply:
👍